samples/transient-clients/Core_6/StockEvents/StockTick.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System;
+
+public class StockTick
```

Comment:
All of the samples just mark is as `IMessage` or `IEvent` if you want it to be explicitly an event.